opengemini/database.go

```diff
@@ -19,7 +19,7 @@ func (c *client) CreateDatabase(database string) error {
 
 	err = queryResult.hasError()
 	if err != nil {
-		return fmt.Errorf("create database err: %s", err)
+		return fmt.Errorf("create database %w", err)
```

Comment:
use delimiter to distinguish error stack

Reply:
@xuthus5 queryPost return an incomplette error msg, and outside addon operation name on it.